javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,72 @@
+const Name = [];
+const toDoList = [];
+
+function getReply(command) {
+  for (i = 0; i < Name.length; i++);
+
+  const NewName = "benjamin";
+  Name.push(NewName);
+
+  if (command.includes("benjamin")) {
+    console.log(`hi my name is ${NewName}`);
+    return `hello  ${NewName} nice to meet you`;
+  } else if (command === "what is my name?") {
+    const anotherName = "claudia";
+    Name.push(anotherName);
+    return `my name is  ${anotherName}`;
+  }
+
+  if (command === "fishing") {
+    for (i = 0; i < toDoList.length; i++) {
+      toDoList.push("fishing");
+      console.log(toDoList);
+      return "Fishing was added to my to do list";
+    }
+  } else if (command === "singing") {
+    for (i = 0; i < toDoList.length; i++) {
+      toDoList.push("singing in the shower");
+      console.log(toDoList);
+      return "singing was added";
+    }
+  } else if (command === "remove") {
```

Comment:
you need to check what the command wants to remove, just because the command has remove doesn't mean that you should remove fishing.